presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/JsonRenderer.java

```diff
@@ -128,6 +133,12 @@ public List<String> getRemoteSources()
         {
             return remoteSources;
         }
+
+        @JsonProperty
+        public List<PlanNodeStatsEstimate> getEstimates()
```

Comment:
yes because this is for deserialize to check for matching estimate which requires a serde for `VariableReferenceExpression`, I will a check in that commit.